packages/experimental/Button/src/ButtonTokens.ts

```diff
@@ -39,7 +37,6 @@ export const defaultButtonTokens: TokenSettings<ButtonTokens, Theme> = () =>
       borderWidth: globalTokens.stroke.width.thin,
       iconSize: 20,
       hasContent: {
-        minWidth: 96,
```

Comment:
Why are we removing the min width?

Reply:
In win32 RN redlined there's no minWidth. There are only paddings:
![image](https://user-images.githubusercontent.com/11574680/145993129-5e7bb999-4021-4012-bced-37b4e92a886d.png)
But I see that in Web readlines they have min width

Reply:
Ah okay. Yeah I think we should keep the minWidth until we can confirm with design that we shouldn't have it in FURN.

Reply:
Yes, I'll ask. I won't merge the PR until it's confirmed

Reply:
I think it's fine to merge it without any changes to minWidth and follow up with another PR later, esp since it's the holidays and it'll probably be a while until you get an answer. Since people can turn off minWidth by just setting it to a small number, I don't think it's a blocker for this PR.